lib/atomic_web/components/button.ex

```diff
@@ -0,0 +1,44 @@
+defmodule AtomicWeb.Components.Button do
+  @moduledoc false
+  use AtomicWeb, :component
```

Comment:
```suggestion
  use AtomicWeb, :live_component
```
You are using `live_patch` 👀